packages/components/icon/README.mdx

```diff
@@ -178,3 +73,11 @@ const CustomIcon = (props) => {
 ### Custom icons
 
 <PropsTable of="Icon" />
```

Comment:
![Screenshot 2022-01-14 at 16 08 01](https://user-images.githubusercontent.com/6597467/149538252-a95134f5-4d89-499a-8cc3-4d7c995f44d2.png)

There's something wrong with the props here, could you take a look?